app/views/hq/orphans/show.html.erb

```diff
@@ -209,6 +209,10 @@
   </div>
   <div class="panel-body">
     <dl class="dl-horizontal">
+      <dt>Partner</dt>
+      <dd><%= @orphan.orphan_list.partner.name %></dd>
```

Comment:
please use `@orphan.partner.name` instead because we have the relation set directly in the model with `has_one :partner, through: :orphan_list, autosave: false`